### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by bytekoders** [arvindr21/slush-wean#5]
I know It might be a bit too much but could you also add some testing framework to any of your scaffoldings?

**Comment by arvindr21** [comment]
Something like https://github.com/nwjs/nw.js/wiki/Testing#how-do-we-test-with-mocha ?

**Comment by bytekoders** [comment]
Yes that would be a great... adding mocha/intern with karma as test runner or maybe something else you might know of... I am still new to the node/node-webkit world so isn't sure of the frameworks are battle tested or not
